# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/reftest.list
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/reftests/filters/reftest.list
--- a/gfx/wr/wrench/reftests/filters/reftest.list	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/reftests/filters/reftest.list	Thu Jun 13 04:43:56 2019 +0000
@@ -1,9 +1,9 @@
 == filter-grayscale.yaml filter-grayscale-ref.yaml
-platform(linux,mac) == draw_calls(6) color_targets(6) alpha_targets(0) filter-blur.yaml filter-blur.png
+platform(linux,mac) == draw_calls(7) color_targets(7) alpha_targets(0) filter-blur.yaml filter-blur.png
 == isolated.yaml isolated-ref.yaml
 == invisible.yaml invisible-ref.yaml
-color_targets(1) alpha_targets(0) == opacity.yaml opacity-ref.yaml
-color_targets(1) alpha_targets(0) == opacity-combined.yaml opacity-combined-ref.yaml
+color_targets(2) alpha_targets(0) == opacity.yaml opacity-ref.yaml
+color_targets(2) alpha_targets(0) == opacity-combined.yaml opacity-combined-ref.yaml
 == opacity-overlap.yaml opacity-overlap-ref.yaml
 == filter-brightness.yaml filter-brightness-ref.yaml
 == filter-brightness-2.yaml filter-brightness-2-ref.yaml
@@ -15,7 +15,7 @@
 == filter-invert.yaml filter-invert-ref.yaml
 == filter-invert-2.yaml filter-invert-2-ref.yaml
 platform(linux,mac) fuzzy(1,133) == filter-large-blur-radius.yaml filter-large-blur-radius.png
-skip_on(android) == draw_calls(4) color_targets(4) alpha_targets(0) filter-small-blur-radius.yaml filter-small-blur-radius.png  # fails on Android emulator and Pixel2
+skip_on(android) == draw_calls(6) color_targets(6) alpha_targets(0) filter-small-blur-radius.yaml filter-small-blur-radius.png  # fails on Android emulator and Pixel2
 == filter-saturate-red-1.yaml filter-saturate-red-1-ref.yaml
 == filter-saturate-red-2.yaml filter-saturate-red-2-ref.yaml
 == filter-saturate-red-3.yaml filter-saturate-red-3-ref.yaml
@@ -28,7 +28,7 @@
 == filter-saturate-red-alpha-1.yaml filter-saturate-red-alpha-1-ref.yaml
 == filter-saturate-green-alpha-1.yaml filter-saturate-green-alpha-1-ref.yaml
 == filter-saturate-blue-alpha-1.yaml filter-saturate-blue-alpha-1-ref.yaml
-== filter-hue-rotate-1.yaml filter-hue-rotate-1-ref.yaml
+fuzzy(1,1) == filter-hue-rotate-1.yaml filter-hue-rotate-1-ref.yaml
 skip_on(android,device) == filter-hue-rotate-alpha-1.yaml filter-hue-rotate-alpha-1-ref.yaml  # Fails on Pixel2
 skip_on(android) fuzzy(1,14) == filter-long-chain.yaml filter-long-chain.png  # fails on Android emulator and Pixel2
 platform(linux,mac) == filter-drop-shadow.yaml filter-drop-shadow.png